src/trees/SerialFunctionTree.cpp

```diff
@@ -479,6 +580,7 @@ template <int D> void SerialFunctionTree<D>::rewritePointers(int nChunks) {
                     int n_ichunk = node->parentSerialIx / this->maxNodesPerChunk;
                     int n_inode = node->parentSerialIx % this->maxNodesPerChunk;
                     node->parent = this->nodeChunks[n_ichunk] + n_inode;
+                    assert(node->parent->serialIx != node->parentSerialIx);
```

Comment:
This assertion fails in some unit tests. Is it not supposed to be `==`?

Reply:
Yes, you are right!
Where can I see where it went wrong? (I tried to get to "details", but did not see this part.

Reply:
First "details", then click on the failing build and scroll to the bottom. There's usually a LOT of scrolling because of verbose build.